src/GitHub.jl

```diff
@@ -1,6 +1,9 @@
 module GitHub
 
 using Compat
+using Nulls
+
+const ?{T} = Union{T, Null}
```

Comment:
This is maybe too cute...

Reply:
I thought we got rid of the ability to do that on master? I think DataArrays calls this union `Data{T}` internally as a stopgap until we get `T?` in Base.

Reply:
If you use `Void` as I suggested, HTTP has [this definition](https://github.com/JuliaWeb/HTTP.jl/blob/f8771109aceefe9f28ce24a6a3f435a58befcf62/src/types.jl#L15):
```julia
const Option{T} = Union{T, Void}
```
then defines internal fields and whatnot in terms of `Option`.